src/middlewares/userMiddlware.js

```diff
@@ -6,6 +6,12 @@ const User = {
     const existingUser = await db.Users.findOne({ where: { email: req.body.email }, raw: true });
     if (existingUser) return sendResult(res, 409, 'This email already exists');
     next();
+  },
+  async checkloginEntries(req, res, next) {
+    if (!req.body.email || !req.body.password || !(req.body.email && req.body.password)) {
```

Comment:
I do not think the last  statement`!(req.body.email && req.body.password)` is necessary here. 
You can also destructure `email` and `password` from `req.body`.

Reply:
Yes